runtime/cere-dev/src/governance/mod.rs

```diff
@@ -0,0 +1,70 @@
+use frame_support::parameter_types;
+use frame_system::EnsureRootWithSuccess;
+
+use super::*;
+
+mod origins;
+pub use origins::{
+	pallet_custom_origins, GeneralAdmin, ReferendumCanceller, ReferendumKiller, Spender,
+	StakingAdmin, Treasurer, WhitelistedCaller,
+};
+mod tracks;
+pub use tracks::TracksInfo;
+
+parameter_types! {
+	pub const VoteLockingPeriod: BlockNumber = 7 * DAYS;
+}
+
+impl pallet_conviction_voting::Config for Runtime {
+	type WeightInfo = pallet_conviction_voting::weights::SubstrateWeight<Runtime>;
+	type RuntimeEvent = RuntimeEvent;
+	type Currency = Balances;
+	type VoteLockingPeriod = VoteLockingPeriod;
+	type MaxVotes = ConstU32<512>;
+	type MaxTurnout =
+		frame_support::traits::tokens::currency::ActiveIssuanceOf<Balances, Self::AccountId>;
+	type Polls = Referenda;
+}
+
+parameter_types! {
+	pub const AlarmInterval: BlockNumber = 1;
+	pub const SubmissionDeposit: Balance = DOLLARS;
```

Comment:
Looks like a very small amount to spam us with malicious referenda

Reply:
It's a `submission` deposit. To initiate an actual referendum in some Track, you need to attach the `decision` deposit. The value for the `decision`  is specific to each track depending on its severity, it's much higher than 1 CERE. As Polkadot [uses the same value](https://github.com/paritytech/polkadot-sdk/blob/release-polkadot-v1.1.0/polkadot/runtime/polkadot/src/governance/mod.rs#L51) in its runtime, I think it's safe.  The `submission`  deposit is common for all proposals and is just the first step to initiate a referenda.

Reply:
Have you agreed with Dan on the [values](https://github.com/Cerebellum-Network/blockchain-node/pull/238/files#diff-64cb60febdc2d008ce9673649727aefbd30cf90f45463cf2bef69eb805262cdaR52)? We have the [old values documented](https://www.notion.so/cere/Governance-Parameters-77d8e7b77e7e421eac9b4b87449259a5) so this may come as a surprise to the one who starts the referenda.